apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/AnyAssertionSamples.kt

```diff
@@ -21,4 +21,164 @@ class AnyAssertionSamples {
             expect(arrayOf(1)).toBe(arrayOf(1))
         }
     }
+
+    @Test
+    fun notToBe() {
+        expect(2).notToBe(3)
+
+        fails {
+            expect(12).notToBe(12)
+        }
+
+        expect(listOf(2)).notToBe(listOf(3))
+
+        fails {
+            expect(listOf(2)).notToBe(listOf(2))
+        }
+    }
+
+    @Test
+    fun isSameAs() {
+        val list = listOf(3)
+        expect(list).isSameAs(list)
+
+        fails {
+            // fails because isSameAs is based on identity, use toBe for equality
+            expect(listOf(3)).isSameAs(listOf(3))
+        }
+    }
+
+    @Test
+    fun isNotSameAs() {
+        expect(listOf(2)).isNotSameAs(listOf(2))
+
+        fails {
+            val list = listOf(3)
+            expect(list).isNotSameAs(list)
+        }
+    }
+
+    @Test
+    fun toBeNullIfNullGivenElse() {
+        expect<Int?>(null).toBeNullIfNullGivenElse(null)
+
+        expect<Int?>(1).toBeNullIfNullGivenElse {
+            toBe(1)
+        }
+
+        fails {
+            // sub-assertion fails
+            expect<Int?>(1).toBeNullIfNullGivenElse {
+                toBe(2)
+            }
+        }
+    }
+
+    @Test
+    fun notToBeNullFeature() {
+        expect<Int?>(1).notToBeNull()
+
+        fails {
+            expect<Int?>(null).notToBeNull()
+        }
+    }
+
+    @Test
+    fun notToBeNull() {
+        expect<Int?>(1).notToBeNull {
+            toBe(1)
+        }
+
+        fails {
+            expect<Int?>(1).notToBeNull {}
+        }
```

Comment:
```suggestion
        fails {
            // because you forgot to define an assertion in the assertion group block
            // use `notToBeNull()` if this is all you want to assert
            expect<Int?>(1).notToBeNull { } 
        }
```